### DISCUSSION TITLE
Mirrormaker2 Oauth2

### DISCUSSION THREAD

**Discussion by towens** [strimzi/strimzi-kafka-operator#4646] [Q&A] [answered]
Hello,

The docs have been very helpful, Just need a little more info to determine a solution.

Context:
  - Two separate k8s clusters i.e. k8s-a k8s-b. In two separate AWS VPC's.
 
We would like the external bootstrap svc in k8s-a to be an oauth app. Is `KafkaClientAuthenticationOAuth` designed for this?
 
The goal:
  - mirror maker deployed on k8s-b. No problem there.
  - mirror maker using the internal bootstrap svc on k8s-b. Clearly no problem there.
  - mirror maker using `KafkaClientAuthenticationOAuth` on k8s-b to auth with k8s-a's external bootstrap svc.

Is this possible?

Thank you!

**Comment by scholzj** [comment, marked as answer]
In the Kafka custom resource, you can have multiple listeners and configure authentication for each of them - and OAuth can be used with the external listeners. So I do not think this should be a problem.

**Reply by towens** [reply]
Appreciate it.
